Applications/CLI/Tests.cmake

```diff
@@ -486,6 +486,47 @@ if(NOT OGS_USE_MPI)
         cube_1e3_expected_pcs_0_ts_101_t_1.000000.vtu cube_1e3_pcs_0_ts_101_t_1.000000.vtu displacement displacement
     )
 
+    # Mechanics; Small deformations, Ehlers (SDE)
+    AddTest(
+        NAME Mechanics_SDE_cube_1e0
```

Comment:
Is there a test for Drucker-Prager model, which can be a simplification of Ehlers and which could be a good example for verification of the implementation.